src/config.ts

```diff
@@ -275,6 +275,9 @@ function normalizeScript(spec: unknown): Script {
   const src = String(script.src);
   const async = script.async === undefined ? false : Boolean(script.async);
   const type = script.type == null ? null : String(script.type);
+  console.warn(
```

Comment:
this shows the warning once for each of the scripts; we could use a different logic to avoid this, but it seemed convenient to squirrel this warning here rather than set up a dedicated function more upstream?